src/boot/util.h

```diff
@@ -12,7 +12,7 @@
 extern uint8_t __executable_start[];
 
 DISABLE_WARNING_REDUNDANT_DECLS;
-void free(void *p);
+void free(void *p); // NOLINT (readability-redundant-declaration)
```

Comment:
`//` is supposed for be for temporary comments. I hope that this indicates that this ugliness is not here to stay ;)

Reply:
Unfortunately it is here to stay ;)